flink-libraries/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -329,14 +341,46 @@ public void stop(SessionContext session) {
 		}
 	}
 
-	private <T> ResultDescriptor executeQueryInternal(ExecutionContext<T> context, String query) {
+	private <C> ProgramTargetDescriptor executeUpdateInternal(ExecutionContext<C> context, String statement) {
+		final ExecutionContext.EnvironmentInstance envInst = context.createEnvironmentInstance();
+
+		// apply update statement
```

Comment:
some of those comments are a little bit unnecessary?

Reply:
On one side yes but on the other side it allows to read the comments from top to bottom and know what the method is doing without having to look at the actual code.

Reply:
But there is no additional value of 
```
// apply update statement
``` 
over 
```
applyUpdate(envInst.getTableEnvironment(), envInst.getQueryConfig(), statement);
```
which is completely self explanatory. This is just form of some small code duplication. Same applies to most of those comments.